Main.java

```diff
@@ -10,6 +12,34 @@
    */
   public static void main(String[] args) {
     // TODO code application logic here
-    
+    Scanner input = new Scanner(System.in);
+
+    //constants for price of each thing
+    double CHARGER_COST = 34.99;
+    double MOTHER_BOARD = 127.50;
+    double MICE = 18.00;
```

Comment:
Make sure that if these are constants that they have the word final in front of them.
Example:
`final double CHARGER_COST = 34.99;`